test/powershell/Modules/Microsoft.PowerShell.Utility/Import-Csv.Tests.ps1

```diff
@@ -90,5 +90,6 @@ Describe "Import-Csv #Type Tests" -Tags "CI" {
         $importTypes = $importObjectList[0].psobject.TypeNames
         $importTypes[0] | Should Be $expectedProcessTypes[0]
         $importTypes[1] | Should Be $expectedProcessTypes[1]
+        $importTypes.Count | Should Be $expectedProcessTypes.Count
```

Comment:
Please move the line before `$importTypes[0]`.

And please add such check in other `It`.